guides/source/generators.md

```diff
@@ -338,7 +338,7 @@ config.generators do |g|
 end
 ```
 
-If you generate another resource, you can see that we get exactly the same result! This is useful if you want to customize your scaffold templates and/or layout by just creating `edit.html.erb`, `index.html.erb` and so on inside `lib/templates/erb/scaffold`.
+If you generate another resource, you can see that we get exactly the same result! This is useful if you want to customize your [scaffold templates](https://github.com/rails/rails/tree/main/railties/lib/rails/generators/erb/scaffold/templates) and/or layout by just creating `edit.html.erb`, `index.html.erb` and so on inside `lib/templates/erb/scaffold`.
```

Comment:
```suggestion
If you generate another resource, you can see that we get exactly the same result! This is useful if you want to customize the [default scaffold templates](https://github.com/rails/rails/tree/main/railties/lib/rails/generators/erb/scaffold/templates) and/or layout by just creating `edit.html.erb`, `index.html.erb` and so on inside `lib/templates/erb/scaffold`.
```

I think the change looks good. Maybe we can update the wording slightly to clarify exactly what we are linking to?